### PR TITLE
Created Generic Storybook Dropdown Element

### LINKED ISSUE #2: EPIC: Refactor Course create/edit to use dropdown for schools
### Discussion

If you go to the course page, and click create you will see a field where you can enter school for the course.

However, we also have a place where we enter information about schools.  We don't want users to be able to put "just anything" in the schools field like they can now.  Instead, they should have a dropdown menu with just the schools that have already been entered into the system. School values are things like UCSB, WSU, OSU, UW-Hilo, etc.

What we want is to "get all" schools from the schools API, build a drop down menu from those results, and use that select from all the already created schools when creating or editing a course.

### Acceptance Criteria

- [ ] As an admin if I go to course page and create a course I should only be able to choose a school from the already created schools in the schools page.

# Implementation Todos

## First Issue: Just create the React component for the dropdown for schools. 

You can look and learn how to create and implement the dropdown from <https://courses.dokku-00.cs.ucsb.edu/>. As you can see if you hover over the quarter you can see the drop-down list.   You can use the storybook to explore how these dropdown work.  

There are also other drop downs, such as the one in the model for personal schedules, that are populated from lists (this is more like what you are bulding.)

In this first issue, JUST focus on creating that one React component, along with fixtures for the schools, storybook story, and tests.   You'll then use it later when refactoring the form.

## Second issue: refactor the form for creating/editing courses to use the dropdown menu instead of a free form text field.

You can go to the courses repo and see how the dropdown is used on the modal for adding courses to a personal schedule, or when searching for courses.

### LINKED ISSUE #25: Create a School Dropdown List element
# User Story

---

* As a user
* I can select schools from a dropdown menu
* That is limited only to the schools provided in the dropdown

### Acceptance Criteria

- [ ] Clicking the input field will reveal a dropdown
- [ ] User can not input in other schools 
- [ ] Dropdown disappears on selection made

### PR DESCRIPTION
**Pull request by ukyl**
### Changes

---

In this PR, we've created a storybook element for a dropdown.

As a User I can select from a dropdown from a (sorted) list of content
dropdownbasic

As a User I can not select anything if there is no list of content provided
dropdownempty

[Link to storybook](https://ucsb-cs156-s24.github.io/proj-organic-s24-5pm-2/prs/32/storybook/?path=/story/components-ouradddropdownform--ten-school-render)

### Issues

---

Partially Closes #2 
Closes #25

### CODE REVIEW

**Review by abhishekambastha04 — DISMISSED** [review]
LGTM

**Review by nicoshinozaki — DISMISSED** [review]
LGTM!

File: src/main/resources/application-production.properties [hunk 1]
```diff
@@ -2,7 +2,8 @@ spring.datasource.url=${JDBC_DATABASE_URL}
 spring.datasource.username=${JDBC_DATABASE_USERNAME}
 spring.datasource.password=${JDBC_DATABASE_PASSWORD}
 
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL9Dialect
+app.showSwaggerUILink=${SHOW_SWAGGER_UI_LINK:${env.SHOW_SWAGGER_UI_LINK:false}}
+spring.h2.console.enabled=${H2_CONSOLE_ENABLED:${env.H2_CONSOLE_ENABLED:false}}
```
Comment (pconrad) [inline src/main/resources/application-production.properties]: This doesn't belong in this PR.  Please revert this file to what's on the main branch.   These other changes should get made in the appropriate PR, *not in this one*.

**Review by pconrad — CHANGES_REQUESTED** [review]
I love this code!  But please remove the unrelated changes.  Then we can get this merged.

File: src/main/resources/application-production.properties (hunk 1 above)
Comment (ukyl) [reply src/main/resources/application-production.properties]: I've removed the file, it came out of my attempt to rebase (I wasn't entirely familiar with how rebasing worked). The file shouldn't be changed on this PR anymore

**Review by ukyl — COMMENTED** [review]
The additional changes have been removed

**Review by pconrad — APPROVED** [review]
LGTM
